bin/pcp-ping

```diff
@@ -4,6 +4,7 @@ require 'pcp/client'
 start_time = Time.now.to_f
 
 Thread.new { EM.run }
+Thread.pass until EM.reactor_running?
```

Comment:
What if the eventmachine's reactor fails to start?
Should we make some effort to avoid the resulting infinite loop here?